src/components/SchedulePanel/components/CourseEvents/index.js

```diff
@@ -0,0 +1,3 @@
+import moment, { now } from 'moment';
```

Comment:
Is this class a placeholder for future features, or is it dead? If the latter, we should delete it.